diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/RulesConfigValidationTest.kt

```diff
@@ -39,7 +40,7 @@ class RulesConfigValidationTest {
             """.trimMargin()
         )
         val exception = assertThrows<IllegalArgumentException> {
-            DiktatRuleSetProvider(file.absolutePath).invoke()
+            DiktatRuleSetFactoryImpl().invoke(DiktatRuleConfigReaderImpl().invoke(file.inputStream()))
```

Comment:
Here and below: up to you, but this can be replaced with

```suggestion
            DiktatRuleSetFactoryImpl()(DiktatRuleConfigReaderImpl()(file.inputStream()))
```

Reply:
ah.. I think construction like `()()` is less readable that `().invoke()`. Than why I tried to create a variable in rest parts. I think better to rewrite the tests here